# main
main_loop_rate: 500
main_duration: 30 # sec
main_setforce_z: 0 # N
# controller
# Teach Mode
main_setpos_z: 400 # 400
stiffness:
  x: 0.02 # N/mm 0.05 ~ 0.25
  y: 0.02
  z: 0.02
comp1:
  k: 0.001 # change to 0.004 to feel excited
  zero: 0 #0.975 #for contact mode
  pole: 0.98
comp2:
  k: 1.0
  zero: 0.0
  pole: 1.0
  limit: 100.0 # (mm) magnitude limit of output  THIS WILL CREATE A BOUNDING BOX

# # Contact Mode
# main_setpos_z: 250 # 400
# stiffness:
#   x: 0.1 # N/mm 0.05 ~ 0.25
#   y: 0.1
#   z: 0.1
# comp1:
#   k: 0.005 # change to 0.004 to feel excited
#   zero: 0.98 #0.975 #for contact mode
#   pole: 0.98
# comp2:
#   k: 1.0
#   zero: 0.0
#   pole: 1.0
#   limit: 100.0 # (mm) magnitude limit of output  THIS WILL CREATE A BOUNDING BOX


forcecontrol_print_flag: true
forcecontrol_file_path: "/home/simplehands/catkin_ws/src/forcecontrol/data/forcecontrol_data.txt"

